library/spdm_requester_lib/libspdm_req_get_key_pair_info.c

```diff
@@ -167,7 +178,40 @@ static libspdm_return_t libspdm_try_get_key_pair_info(libspdm_context_t *spdm_co
         status = LIBSPDM_STATUS_INVALID_MSG_SIZE;
         goto receive_done;
     }
-    spdm_response_size = sizeof(spdm_key_pair_info_response_t) + spdm_response->public_key_info_len;
+
+    rsp_pqc_asym_algo_capabilities = 0;
+    rsp_current_pqc_asym_algo = 0;
+    if (spdm_response->header.spdm_version >= SPDM_MESSAGE_VERSION_14) {
+        if (spdm_response_size < sizeof(spdm_key_pair_info_response_t) +
+            spdm_response->public_key_info_len + sizeof(uint32_t)) {
+            status = LIBSPDM_STATUS_INVALID_MSG_SIZE;
+            goto receive_done;
+        }
+        ptr = (uint8_t *)spdm_response + sizeof(spdm_key_pair_info_response_t) + spdm_response->public_key_info_len;
+        pqc_asym_algo_cap_len = libspdm_read_uint32 (ptr);
```

Comment:
The size of `PqcAsymAlgoCapLen` is one byte.

Reply:
fixed.